app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+  include Pagy::Frontend
+  def pagy_url_for(page, pagy, url=false)
```

Comment:
what is this method for? Add code comment above this helper method

Reply:
Added the comment. This helper method is overriding a method of pagy gem to provide facility of having a custom base link instead of request.path for the pagination links

Reply:
This is an internal helper for pagy. I am redefining it to be able to add custom base path to pagy links. I have added the comment now.